.github/rpm/twilio.spec

```diff
@@ -0,0 +1,50 @@
+# Spec file is needed to build a rpm package
```

Comment:
SPEC file provides necessory details required to build an rpm package. It contains from description of the package to how the files would be places during install.